libs/common/src/middleware/front-end-error-handler.handle-csrf-token-error.test.ts

```diff
@@ -0,0 +1,28 @@
+import httpMocks from 'node-mocks-http';
+import { Response } from 'express';
+import { handleCsrfTokenError } from './front-end-error-handler';
+
+describe('front-end-error-handler', () => {
+  console.error = jest.fn();
+  let res: httpMocks.MockResponse<Response>;
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    ({ res } = httpMocks.createMocks());
+  });
```

Comment:
New line please 😛

Reply:
Just checking you were paying attention :D added, thanks